contrib/BBEditTextWrangler-julia.plist

```diff
@@ -843,10 +843,9 @@
               <string>rationalize</string>
               <string>read</string>
               <string>read!</string>
-              <string>readall</string>
+              <string>readstring</string>
```

Comment:
See #8298 - "It was more of a hack and slash effort than a script. I got a list of names by calling names() in the REPL and copy/paste -ing. Perhaps I'll do an automated version one day..."

Every other renaming and refactoring that has touched this file has kept it sorted. That's easier to figure out from a diff than ignoring review and adding work to clean it up on master. Ref https://github.com/JuliaLang/julia/pull/9468#issuecomment-72445138 where this has happened before.